<launch>
  <arg name="UAV_NAME" default="$(optenv UAV_NAME)" />


  <group ns="$(arg UAV_NAME)">
    <node pkg="uav_checkpoints_nav" type="grab_goals.py" name="grab_goals" output="screen">
        <rosparam command="load" file="$(find uav_checkpoints_nav)/config/simple.yaml" />
    </node>
  </group>

</launch>
